dex-load/src/main/scala/com/wavesplatform/dex/load/WavesDexLoadCli.scala

```diff
@@ -179,90 +183,123 @@ object WavesDexLoadCli extends ScoptImplicits {
               case Command.Check =>
                 implicit val system = ActorSystem()
 
-                cli.log[Id](
-                  s"""Arguments:
-                     |  DEX REST API            : ${args.dexRestApi}
-                     |  DEX WebSocket API       : ${args.dexWsApi}
-                     |  Feeder file             : ${args.feederFile.getAbsolutePath}
-                     |  Accounts number         : ${args.accountsNumber}
-                     |  Collect time            : ${args.collectTime}
-                     |  WebSocket response time : ${args.wsResponseWaitTime}\n""".stripMargin
-                )
-
-                val clients: Seq[WsCollectChangesClient] = cli.wrapByLogs[Id, Seq[WsCollectChangesClient]]("Creating clients.. ") {
-                  WsAccumulateChanges.createClients(args.dexWsApi, args.feederFile, args.accountsNumber)
+                def checkLeaps() = {
+                  val clients: Seq[WsCollectChangesClient] =
+                    WsAccumulateChanges.createClients(args.dexWsApi, args.feederFile, args.accountsNumber, args.wsCheckType)
+
+                  val withLeaps =
+                    try {
+                      Future.traverse(clients)(_.run())
+                      Await.result(GlobalTimer.instance.sleep(args.collectTime), Duration.Inf)
```

Comment:
We can use `Thread.sleep` here. `GlobalTimer.instance.sleep` is used in asynchronous code with `Futures` :)